server/aws-lsp-codewhisperer/src/client/streamingClient/codewhispererStreamingClient.ts

```diff
@@ -4,7 +4,7 @@ import { readFileSync } from 'fs'
 
 const codeWhispererRegion = 'us-east-1'
 
-const codeWhispererEndpoint = 'https://codewhisperer.us-east-1.amazonaws.com/'
+const codeWhispererEndpoint = process?.env.AWS_Q_ENDPOINT_URL ?? 'https://codewhisperer.us-east-1.amazonaws.com/'
```

Comment:
How will we define those in browser?

Reply:
process will be undefined in browser and this will fall back to the default url.

Reply:
if we wanted to, we could use webstorage/cookies or [self](https://developer.mozilla.org/en-US/docs/Web/API/Window/self) to override the default endpoint URL

Reply:
We could inject env vars through `runtime` feature, as it's currently done for `platform`.